From d145a456d9fcdc5d8560f4035bb6e35531ba8493 Mon Sep 17 00:00:00 2001
From: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date: Tue, 20 Sep 2016 22:24:13 +0200
Subject: [PATCH] mach: Fix old-style function definition.

	* mach/mach_error.c (mach_error): Fix old-style function definition.
---
diff --git a/mach/mach_error.c b/mach/mach_error.c
index 8bbb2ec74e..0d3082d021 100644
--- a/mach/mach_error.c
+++ b/mach/mach_error.c
@@ -63,9 +63,7 @@
 extern char * mach_error_string_int(mach_error_t, boolean_t *);
 
 void
-mach_error( str, err )
-	char	*str;
-	mach_error_t		err;
+mach_error(char *str, mach_error_t err)
 {
 	char * err_str;
 	char buf[1024];
